systers_portal/templates/meetup/members.html

```diff
@@ -0,0 +1,11 @@
+{% extends "meetup/base.html" %}
+
+{% block meetup_location_page_content %}
+    <ul>
+     {% for member in object.members.all %}       
+            <li>
+                <a href="{{ member.homepage_url }}"> {{ member }}</a> 
```

Comment:
I'd rather use `get_absolute_url`. There are some examples in code.
Again no need for extra whitespace.